src/marketplace/items/item-details.tsx

```diff
@@ -0,0 +1,228 @@
+import * as React from 'react';
+import injectSheet, { StyleSheet, ClassNameMap } from 'react-jss';
+
+import { Grid, Divider, FormGroup, FormControl } from '@material-ui/core'
+import { H2, H3 } from '../../typography/headings';
+import { P } from '../../typography/paragraph';
+import { TickIcon } from '../../icons/tick';
+import { UnlockIcon } from '../../icons/unlock';
+import { StyledButton } from '../../common/styled-button';
+
+const styles: StyleSheet = {
+  root: {
+    width: '946px',
+    height: '100%',
+    marginTop: '30px',
+    marginBottom: '30px',
+    border: 'solid 1px #303c49',
+    borderRadius: '4px'
+  },
+
+  title: {
+    margin: '20px'
+  },
+
+  icon: {
+    marginLeft: '20px'
+  },
+
+  header: {
+    backgroundColor: '#2a3540'
+  },
+
+  body: {
+    textAlign: 'justify',
+    margin: '20px',
+    color: '#fff',
+    fontFamily: 'ProximaNovaRegular',
```

Comment:
We really should specify fallbacks for font family: 
'ProximaNovaRegular, Arial, Helvetica, sans-serif'

Reply:
@sk91 makes sense, I will actually get font details from the common css so we can define there once for all.